core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -521,7 +522,8 @@ private Certificate fetchEphemeralCertificate(KeyPair keyPair) {
       try {
         credentials.get().refresh();
         String token = credentials.get().getAccessToken().getTokenValue();
-        request.setAccessToken(token);
+        String strippedToken = CharMatcher.is('.').trimTrailingFrom(token);
```

Comment:
Should we add a brief comment above or a TODO to remove this once we fix wherever this is coming from?